frontend/src/pulldasher/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { usePulls } from './pulls-context';
+import { Pull } from '../pull';
+import { Navbar } from '../navbar';
+import { Column } from '../column';
+import { Box, SimpleGrid } from "@chakra-ui/react"
+
+export const Pulldasher: React.FC = function() {
+   const pulls: Pull[] = usePulls();
+   const pullsCIBlocked = pulls.filter(pull => pull.isCiBlocked());
+   const pullsDeployBlocked = pulls.filter(pull => pull.isDeployBlocked());
+   const pullsReady = pulls.filter(pull => pull.isReady());
+   const pullsDevBlocked = pulls.filter(pull => pull.isDevBlocked());
+   const pullsNeedingCR = pulls.filter(pull => !pull.isCrDone());
+   const pullsNeedingQA = pulls.filter(pull => !pull.isQaDone());
+   return (<>
+      <Navbar mb={4}/>
+      <Box maxW={1024} m="auto" px="var(--body-gutter)">
+      <SimpleGrid columns={3} spacing="var(--body-gutter)">
```

Comment:
Should this be
```suggestion
      <SimpleGrid columns={2} spacing="var(--body-gutter)">
```

Reply:
Yes, I'll fix it when I actually implement those leaderboards.